docs/faq.rst

```diff
@@ -1255,6 +1276,20 @@ It may be useful to set ``BORG_RELOCATED_REPO_ACCESS_IS_OK=yes`` to avoid the
 prompts when renaming multiple repositories or in a non-interactive context
 such as a script. See :doc:`deployment` for an example.
 
+My backup disk is full, what can I do?
+--------------------------------------
+
+First try to run :ref:`borg_check` to find out if there is any error in your
+backup repository. If something is wrong, first try to repair it before running
+any other Borg command.
+
+Second, if you ran a backup that has stopped due to a full disk, there may be a
+checkpoint archive that you can delete and get disk space back running
+:ref:`borg_delete` and :ref:`borg_compact` afterwards. Run :ref:`borg_list` with
+``--consider-checkpoints`` to show checkpoint archives.
```

Comment:
Even deleting objects in the repo needs disk space (because it appends "DEL chunkid" entries and a commit), so this only works if there is still some free space.